README.md

```diff
@@ -48,12 +48,17 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 ## Running the website locally using Hugo
 
-Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run Hugo directly (not inside a container). You need to install the version ofHugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
 
 To build and test the site locally, run:
 
 ```bash
 make serve
+
```

Comment:
What do you think about listing the different commands in separate code blocks or list only
`hugo server --buildFuture`? The `make serve` target performs an additional check.

Reply:
That extra check from `make serve` is useful and if available I'd steer readers to use it.